### PR TITLE
Add "object" term in timer action for object

### LINKED ISSUE #2999: Object timers should have the word "object" in the description
## Description

When trying to use a timer, users can easily get confused between a scene timer and an object timer.  One way to improve this is by making it obvious which type of timer is being referenced.  Scene timers already have the text "scene" in the description, but object timers do not.  See screenshot:

Screenshot 2021-09-01 103926

## Solution suggested

**Describe the solution**

Add the word "object" to descriptions of all object timer actions and conditions (and perhaps expressions?)

### PR DESCRIPTION
**Pull request by Bouh**
Close #2999

### CODE REVIEW

**Comment by tristanbob** [comment]
Can you also add the conditions for object timers?

**Comment by Bouh** [comment]
> Can you also add the conditions for object timers?

Done, please check the changes and let me know :)

File: Core/GDCore/Extensions/Builtin/BaseObjectExtension.cpp [hunk 1]
```diff
@@ -795,8 +795,8 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsBaseObjectExtension(
 
   obj.AddCondition(
          "ObjectTimer",
-         _("Value of a timer"),
-         _("Test the elapsed time of a timer."),
+         _("Value of a timer object"),
```
Comment (4ian) [inline Core/GDCore/Extensions/Builtin/BaseObjectExtension.cpp:798]: Everywhere: 
* "a timer object" => "an object timer"
* "Timer object" => "Object timer"

File: Core/GDCore/Extensions/Builtin/BaseObjectExtension.cpp [hunk 2]
```diff
@@ -828,8 +828,8 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsBaseObjectExtension(
       .AddParameter("string", _("Timer's name"));
 
   obj.AddAction("PauseObjectTimer",
-                _("Pause a timer"),
-                _("Pause a timer."),
+                _("Pause an object timer"),
+                _("Pause an object timer."),
```
Comment (4ian) [inline Core/GDCore/Extensions/Builtin/BaseObjectExtension.cpp:832]: We should also do that for UnPauseObjectTimer, RemoveObjectTimer and the ObjectTimerElapsedTime expression ;)

**Review by 4ian — APPROVED** [review]

**Pull request merged**